apps/condo/lang/en/en.json

```diff
@@ -1697,6 +1697,9 @@
   "banking.category.THERMAL_ENERGY.name": "Thermal energy",
   "banking.category.COLD_WATER_SUPPLY.name": "Cold water supply",
   "banking.category.ELECTRIC_ENERGY.name": "Electric energy",
+  "banking.category.INCOME.name": "Income payments",
+  "banking.costItem.INCOME_FROM_RESIDENT.name": "Income from resident",
+  "banking.costItem.INCOME_FROM_LEGAL.name": "Income from legal entities",
   "banking.costItem.EMERGENCY_SERVICE.name": "Emergency service",
   "banking.costItem.RENT.name": "Rent",
   "banking.costItem.BANKING_SERVICES.name": "Banking services",
```

Comment:
```suggestion
  "banking.costItem.BANKING_SERVICES.name": "Banking service",
```

Reply:
Are we using plural or singular form for all cost items?
I think, plural fits better, because it will be displayed in a chart that accumulates set of transactions.